python/ffsim/tenpy/circuits/gates.py

```diff
@@ -0,0 +1,315 @@
+import numpy as np
```

Comment:
Since we are simulating fermions, we should define gates with fermionic names rather than their qubit equivalents. The natural gates for fermions are slightly different, and can be found in [this file](https://github.com/qiskit-community/ffsim/blob/main/python/ffsim/gates/basic_gates.py). If I'm not mistakaen, the approximate equivalents would be
- `xy` -> `apply_givens`
- `phase` -> `apply_num_interaction`
- `cphase1` -> `apply_on_site_interaction`